core-test/src/main/java/org/openstack4j/api/AbstractTest.java

```diff
@@ -53,7 +54,12 @@ protected void startServer() throws UnknownHostException {
 
         InetAddress inetAddress = InetAddress.getByName("localhost");
         LOG.info("localhost inet address: " + inetAddress.toString());
-        LOG.info("Tests using connector: " + HttpExecutor.create().getExecutorName() + " on " + getHost());
+
+        try {
+            LOG.info("Tests using connector: " + HttpExecutor.create().getExecutorName() + " on " + getHost());
+        } catch (ConnectorNotFoundException ex) {
+            LOG.info("Tests without any connector");
```

Comment:
I have not retested this change. Is this working both in IDE and maven?

Reply:
It allows test that connector is not needed, such as 'VolumeSnapshotTests#createVolumeSnapshotV2', both IDE and maven works.  If test like 'VolumeSnapshotTests#listVolumeSnaphotsV1' that connector is required, both IDE and maven failed in module`core-test`, and `ConnectorNotFoundException`  still be thrown by `HttpConnector.execute` (behaves the same as before). It won't affect whole building process. And why the IDE and maven failed will be another question. See #231 for details.

Reply:
Thanks, makes sense now!